File: DayTwo/JavaScript/AfterBreak/fire_drill_one.js
let getAge = (sonsAge,fathersAge)=>{
    if(sonsAge < 0 || fathersAge <= 0) return;
    if(sonsAge > fathersAge) return;

    let twiceTheAge = fathersAge % sonsAge
    let twice = sonsAge * 2
    let difference = fathersAge - twice;

    if(twiceTheAge + fathersAge == sonsAge * 2)
        console.log(`father will be twice sons age in ${twiceTheAge} years time`)
    else if(twice > fathersAge){
       console.log(`father will be twice sons age in ${difference * - 1} years time`);
    }
    else
        console.log(`father was twice sons age in ${fathersAge - (sonsAge * 2)} years ago `)

}





getAge(60,16);

File: DayTwo/Python/AfterBreak/fire_drill_one.py
def getAge(sonsAge,fathersAge):
    if(sonsAge < 0 or fathersAge <= 0): return
    if(sonsAge > fathersAge): return
        
    twiceTheAge = fathersAge % sonsAge
    twice = sonsAge * 2
    difference = fathersAge - twice

    if(twiceTheAge + fathersAge == sonsAge * 2):
        print(f"father will be twice sons age in {twiceTheAge} years time ")
    elif (twice > fathersAge):
        print(f"father will be twice sons age in {difference * - 1} years time ");
    
    else:
        print(f"father was twice sons age in {fathersAge - (sonsAge * 2)} years ago ")







getAge(60,76);


